src/app/shared/components/message-table/message-table.component.html

```diff
@@ -30,7 +31,11 @@
           <td>{{message.protocol}}</td>
         </tr>
       </table>
-    </div>        
+
+      <div *ngIf="messages.length == 0" class="no-msg">
```

Comment:
Maybe we should use `ngSwitch` when there are more than 2 cases.

Reply:
I agree, we can do it for `mode`. For `messages.length` there is a single case.